AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_DirectionalLightAdded.py

```diff
@@ -74,13 +224,40 @@ def AtomEditorComponents_DirectionalLight_AddedToEntity():
     5) Enter/Exit game mode.
     6) Test IsHidden.
     7) Test IsVisible.
```

Comment:
we should have Enter/Exit game mode occur after the component has been configured in some way otherwise this misses the shadow camera and other connected component interactions.

Reply:
You're right, but damnit now I need to re-number everything :P 

Fixed!

Reply:
This was not changed by my code, and has nothing to do with my PR